lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/address/api/handler/IssueCredentialHandlerTest.java

```diff
@@ -248,9 +252,11 @@ void shouldThrowAWSExceptionWhenAServerErrorOccursDuringRetrievingAnAddressItemW
         verify(mockAddressService).getAddressItem(sessionId);
         verify(mockEventProbe).counterMetric(ADDRESS_CREDENTIAL_ISSUER, 0d);
         verify(mockAuditService, never()).sendAuditEvent(any(AuditEventType.class));
-        String responseBody = new ObjectMapper().readValue(response.getBody(), String.class);
+        var responseBody = new ObjectMapper().readValue(response.getBody(), Map.class);
```

Comment:
Using Map.class here results in a "Raw use..." warning. We should be able to use:
Map<String, Object> responseBody =
                objectMapper.readValue(response.getBody(), new TypeReference<>() {});